Sources/SwiftCli/main.swift

```diff
@@ -31,7 +30,7 @@ struct StartProgram: ParsableCommand {
 
         let outputWriter: OutputWriter
         if let outputFile = outputFile {
-            outputWriter = FileWriter(filePath: "Charts", fileName: outputFile)
+            outputWriter = FileWriter(fileNameAndPath: outputFile)
```

Comment:
I would just call it `filePath`. Usually we assume file path includes the file name.